packages/framework/src/Model/Order/Order.php

```diff
@@ -1138,11 +1138,11 @@ public function fillOrderProducts(
 
             if ($quantifiedItemDiscount !== null) {
                 $name = sprintf(
-                   '%s %s - %s',
-                   t('Promo code', [], 'messages', $locale),
-                   $numberFormatterExtension->formatPercent(-$orderPreview->getPromoCodeDiscountPercent(), $locale),
-                   $orderItem->getName()
-               );
+                    '%s %s - %s',
```

Comment:
these changes do not belong to this commit, right? :innocent:

Reply:
This will be changed in previous commit because it was moved to protected method